pontoon/base/management/commands/calculate_stats.py

```diff
@@ -34,20 +32,9 @@ def handle(self, *args, **options):
             "disabled", "resource_count"
         )
 
-        for index, project in enumerate(projects):
-            log.info(
-                'Calculating stats for project "{project}" ({index}/{total})'.format(
-                    index=index + 1,
-                    total=len(projects),
-                    project=project.name,
-                )
-            )
-
-            translated_resources = TranslatedResource.objects.filter(
-                resource__project=project
-            )
-
-            for translated_resource in translated_resources:
-                translated_resource.calculate_stats()
```

Comment:
I don't think we've been using this command recently (i.e. in the last couple of years). Should we remove after a while if issues with that stats stop resurfacing?

Reply:
Probably, as a parts of the aggregated-stats refactor.